app/models/waste_carriers_engine/key_person.rb

```diff
@@ -39,5 +40,14 @@ def set_date_of_birth
     rescue TypeError
       errors.add(:dob, :invalid_date)
     end
+
+    def set_individual_dob_fields
+      return if dob_year.present? & dob_month.present? & dob_day.present?
```

Comment:
Any particular reason why choosing the binary operator over the logical one? Just curious :D

Reply:
No idea, just tired I think! I'll change it

Reply:
Well, since `present?` returns true/false it would have been fine :D But yes, better to be consistent :D